SWLOR.Game.Server/Feature/ShipModuleDefinition/MiningLaserModuleDefinition.cs

```diff
@@ -132,6 +138,7 @@ private void MiningLaser(string itemTag, string name, string shortName, int requ
                             var xp = Skill.GetDeltaXP(delta);
 
                             Skill.GiveSkillXP(activator, SkillType.Piloting, xp);
+                            SetLocalBool(target, "BEING_MINED", false);
```

Comment:
I'd put this variable outside the PC check. If, somehow, a non-PC uses a mining laser the rock will be locked forever.